### PR TITLE
docs: xlinectl document

### PR DESCRIPTION
**Pull request by bsbds**
Please briefly answer these questions:

* what problem are you trying to solve? (or if there's no problem, what's the motivation for this change?)

    add document to xlinectl.

* what changes does this pull request make?

* are there any non-obvious implications of these changes? (does it break compatibility with previous versions, etc)

### CODE REVIEW

File: xlinectl/README.md [hunk 1]
```diff
@@ -1,3 +1,138 @@
...
+    ```
+- wait_synced_timeout <TIMEOUT> -- The timeout for Curp client waiting synced(in secs) [default: 2]
+- propose_timeout <TIMEOUT> -- The timeout for Curp client proposing request(in secs) [default: 1]
+- retry_timeout <TIMEOUT> -- The timeout for Curp client retry interval(in millis) [default: 50]
+- printer_type <TYPE> -- The format of the result that will be printed [default: SIMPLE] [possible values: SIMPLE, FIELD]
+
+## Output Format
+
+All command output will first print the response header, and then print the response fields
+
```
Comment (iGxnon) [inline xlinectl/README.md:29]: I think we can provide an output format example.

File: xlinectl/README.md [hunk 2]
```diff
@@ -1,3 +1,138 @@
...
+    ```
+- password <PASSWD> -- The password of the user, should exist if password not set in `--user`
+    ```bash
+    # connect to servers using user `foo` with password `bar`
+    ./xlinectl --user foo --password bar
+    ```
+- wait_synced_timeout <TIMEOUT> -- The timeout for Curp client waiting synced(in secs) [default: 2]
+- propose_timeout <TIMEOUT> -- The timeout for Curp client proposing request(in secs) [default: 1]
+- retry_timeout <TIMEOUT> -- The timeout for Curp client retry interval(in millis) [default: 50]
+- printer_type <TYPE> -- The format of the result that will be printed [default: SIMPLE] [possible values: SIMPLE, FIELD]
```
Comment (Phoenix500526) [inline xlinectl/README.md:24]: We should add some output format examples here. Does the FIELD type look like JSON format?

File: xlinectl/README.md [hunk 3]
```diff
@@ -1,3 +1,138 @@
...
+    ```bash
+    # connect to servers using user `foo` with password `bar`
+    ./xlinectl --user foo:bar
+    ```
+- password <PASSWD> -- The password of the user, should exist if password not set in `--user`
+    ```bash
+    # connect to servers using user `foo` with password `bar`
+    ./xlinectl --user foo --password bar
+    ```
+- wait_synced_timeout <TIMEOUT> -- The timeout for Curp client waiting synced(in secs) [default: 2]
```
Comment (Phoenix500526) [inline xlinectl/README.md:21]: Does TIMEOUT need to specify the unit?  `--wait_synced_timeout 2s` and `--wait_synced_timeout 2`, which one is correct?

File: xlinectl/README.md (hunk 1 above)
Comment (bsbds) [reply xlinectl/README.md:40]: added

File: xlinectl/README.md (hunk 2 above)
Comment (bsbds) [reply xlinectl/README.md:34]: I added the SIMPLE format document. The FIELD type is more like a human readable format, and too verbose to put in document.

File: xlinectl/README.md (hunk 3 above)
Comment (bsbds) [reply xlinectl/README.md:28]: no, it should be consistent with the default type.

**Review by iGxnon — APPROVED** [review]

**Review by Phoenix500526 — APPROVED** [review]

**Pull request merged**